modules/ROOT/pages/deployment/wopi/wopi.adoc

```diff
@@ -0,0 +1,103 @@
+= Office Applications Using WOPI
+:toc: right
+:wopi_subdir: /ocis_wopi
+:download-gh-directory-url: https://download-directory.github.io
+:traefik_url: https://traefik.io/traefik/
+:nginx_url: https://www.nginx.com
+:apache_url: https://www.apache.org
+
+:description: Infinite Scale uses the WOPI protocol to integrate office applications. This document describes the general flow and basic settings used.
+
+== Introduction
+
+{description} For more details on possible configuration values see the xref:conf-examples/office-integration.adoc[Office Integration] document at the configuration examples. 
+
+For the time being, the supported office applications are:
+
+* Collabora
+* OnlyOffice
```

Comment:
What about MS Office 365?

Reply:
I have been told by Willy, as far I remember, that only the above are supported. And I know from the past, that MS Office 365 support is "special". Lets have a chat/video about that. Note that we can add the line item with MS Office at any time afterwards.

Reply:
Also see the [ownCloud Server](https://doc.owncloud.com/server/10.11/admin_manual/enterprise/collaboration/msoffice-wopi-integration.html#introduction) docs:

...If you want to integrate the [Office 365 (cloud)](https://products.office.com/en-us/business/office) version of Microsoft Office Online, you need to [get in touch with us](https://owncloud.com/support)...

And the comment in [Configuring the WOPI App in ownCloud](https://doc.owncloud.com/server/10.11/admin_manual/enterprise/collaboration/msoffice-wopi-integration.html#configuring-the-wopi-app-in-owncloud)

Reply:
I have an idea how to integrate O365 into this document, but I will file an additional PR.